ynput/ayon-core
# Fusion: Allow render publishes to have custom frame range per saver

## Changelog Description

Allow render publishes to have custom frame range per saver

## Additional info

Screenshot:
![image](https://github.com/ynput/ayon-core/assets/2439881/c0305336-0e19-4cbd-b80b-72dfe725edda)

There is currently no way to hide the 'custom frame range' attributes in the UI if the frame range source was not set to Custom Frame Range so it will always (confusingly) be shown. I know that is an issue @antirotor and @iLLiCiTiT are aware of but is currently in the backlog. I believe https://github.com/ynput/ayon-core/issues/134 is part of that as a tracked issue.

---

Note: It may potentially be that this does not work correctly for farm submitted renders - but if that's the case then I suspect that's also originally an issue with mixing the other "frame range sources" if they were different frame ranges. If that's indeed not working as intended, then that'd be a separate issue from this PR's functionality so would be great if you could report that as a separate issue if that's the case.

## Testing notes:

1. Create a few saver instances, say five different ones.
2. Set them to different Frame Range Source
3. Set two of them to "Custom frame range"
4. Set different custom frame ranges 

Each of the instances should render and publish only the specified frame ranges.
